src/components/About/ProfileCard.module.css

```diff
@@ -0,0 +1,87 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  padding-top: 2rem;
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+  height: 13.813rem;
+  width: 18.875rem;
+  background: var(--dark-theme-background);
+}
+
+.avatar {
+  width: 96px;
+  height: 96px;
+  border-radius: 50%;
+  border-color: #C4FFFE;
+  border-width: 2px;
+  border-style: solid;
+  margin-bottom: 1rem;
+}
+
+.name {
+  color: var(--rapture-blue);
+  width: 100%;
+  text-align: center;
+}
+
+.title {
+  color: var(--primary-white);
+  font-style: italic;
+  font-weight: 500;
+  width: 100%;
+  text-align: center;
+}
+
+.quote {
+  margin-top: 0.5rem;
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+  text-align: center;
+  color: var(--primary-white);
+  font-weight: 600;
+  margin-bottom: 1.5rem;
+}
+
+.showButton, 
+.hideButton {
+  position: absolute;
+  bottom: -28px;
+  left: 50%;
+  transform: translate(-50%);
```

Comment:
Lines 57 and 58 are perfectly cancelling each other out—try disabling them both in the browser to see it—so you can delete those two lines.

Reply:
I removed them.